src/components/quote/code/views/quoteView.jsx

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+module.exports = React.createClass({
+
+  propTypes: {
+    purpose: React.PropTypes.oneOf(['default', 'primary', 'secondary', 'success', 'warning', 'danger', 'info']),
```

Comment:
The purposes and sizes look to be duplicated in nearly all of the components, is there any reason this couldn't be moved to a shared location? (not as part of this PR but in the future)